workspaces/sonarqube/plugins/sonarqube/src/components/SonarQubeCard/SonarQubeCard.tsx

```diff
@@ -14,6 +14,22 @@
  * limitations under the License.
  */
 
+/*
```

Comment:
This can be removed as it's repeated now

Reply:
Thanks for pointing it.
Done:)